.github/workflows/link-rot-checker.yml

```
@@ -0,0 +1,25 @@
+name: Link-rot-Checker
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+  - cron: "00 18 * * *"
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+    - uses: actions/checkout@v4
+    - name: Link Checker
+      id: lychee
+      uses: lycheeverse/lychee-action@v2
+      with:
+        fail: false
```

Let's add an explicit output:
```suggestion
        fail: false
        output: ./lychee/out.md
```

---

Done.